client/components/logged-out-form/footer.scss

```diff
@@ -1,3 +1,5 @@
+@import "../../assets/stylesheets/shared/mixins/elevation";
```

Comment:
Storybook build of this individual file fails without explicit imports for this dependency